_docs/tasks/request-timeouts.md

```diff
@@ -0,0 +1,120 @@
+timeout to 1 second.
+To see its effect, however, we'll also introduce an artificial 2 second delay in calls
+to the `ratings` service.
+ 
+1. Route requests to v2 of the `reviews` service, i.e., a version that calls the `ratings` service
+
+   ```bash
+   $ cat <<EOF | istioctl replace
+   type: route-rule
+   name: reviews-default
+   spec:
+     destination: reviews.default.svc.cluster.local
+     route:
+     - tags:
+         version: v2
+   EOF
+   ```
+
+1. Add a 2 second delay to calls to the `ratings` service:
+
+   ```bash
+   $ cat <<EOF | istioctl replace
+   type: route-rule
+   name: ratings-default
+   spec:
+     destination: ratings.default.svc.cluster.local
+     route:
+     - tags:
+         version: v1
+     httpFault:
+       delay:
+         percent: 100
+         fixedDelaySeconds: 2
+   EOF
+   ```
+
+1. Open the Bookinfo URL (http://$GATEWAY_URL/productpage) in your browser
+
+   You should see the bookinfo application working normally (with ratings stars displayed),
+   but there is a 2 second delay whenever you refresh the page.
+
+1. Now add a 1 second request timeout for calls to the `reviews` service
+   
+   ```bash
+   $ cat <<EOF | istioctl replace
+   type: route-rule
+   name: reviews-default
+   spec:
+     destination: reviews.default.svc.cluster.local
+     route:
+     - tags:
+         version: v2
+     httpReqTimeout:
+       simpleTimeout:
+         timeoutSeconds: 1
+   EOF
+   ```
+
+1. Refresh the Bookinfo web page
+
+   You should now see that it returns in 1 second (instead of 2), but the reviews are unavailable.
+
+
+## Understanding what happened
+
+In this task, we used Istio to set the request timeout for calls to the `reviews`
+microservice to 1 second (instead of the default 15 seconds). 
+Since the `reviews` service subsequently calls the `ratings` service when handling requests,
+we used Istio to inject a 2 second delay in call to `ratings`, so that we would cause the
+`reviews` service to take longer than 1 second to complete and consequently we coud see the
+timeout in action. 
+
+We observed that the Bookinfo productpage (which calls the `reviews` service to populate the page),
+instead of displaying reviews, displayed
+the message: Sorry, product reviews are currently unavailable for this book.
+This was the result of it recieving the timeout error from the `reviews` service.
+
+If you check out the [fault injection task](./fault-injection.html), you'll find out that the `productpage`
+microservice also has its own application-level timeout (3 seconds) for calls to the `reviews` microservice.
+Notice that in this task we used an Istio route rule to set the timeout to 1 second.
+Had we instead set the timeout to something greater than 3 seconds (e.g., 4 seconds) the timeout
+would have had no effect since the more restrictive of the two will take precedence.
+More details can be found [here](/docs/concepts/traffic-management/handling-failures.html#faq).
+
```

Comment:
May be we could move the text about per-request timeout (using http headers) into this page? like Additional information or whatever.. Same applies to retries.